NYPLCardCreator/Helpers/NYPLColor.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  NYPLColor.swift
+//  NYPLCardCreator
+//
+//  Created by Ernest Fan on 2021-09-24.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import UIKit
+
+private enum ColorAsset: String {
+  case primaryBackground
+  case primaryText
+  case action
+  case disabledFieldText
+}
+
+class NYPLColor {
+  static var primaryBackgroundColor: UIColor {
+    if #available(iOS 13.0, *),
+       UIScreen.main.traitCollection.userInterfaceStyle == .light {
+      return .systemBackground
+    } else if #available(iOS 11.0, *),
+       let color = UIColor(named: ColorAsset.primaryBackground.rawValue) {
+      return color
+    }
+
+    return .white
+  }
+  
+  static var primaryTextColor: UIColor {
+    if #available(iOS 13.0, *),
+       UIScreen.main.traitCollection.userInterfaceStyle == .light {
+      return .label
+    } else if #available(iOS 11.0, *),
+       let color = UIColor(named: ColorAsset.primaryText.rawValue) {
+      return color
+    }
+
+    return .black
+  }
+  
+  static var actionColor: UIColor {
+    if #available(iOS 11.0, *),
```

Comment:
should we do the same logic here too for iOS 13, similar to what you did for `primaryBackgroundColor` ?  iOS 13 provides the `.link` adaptable color. Also we were using that before (see IntroductionViewController.swift)

Reply:
Done